library/jetty

```diff
@@ -1,5 +1,5 @@
 Maintainers: Mike Dillon <mike@appropriate.io> (@md5),
-             Greg Wilkins <gregw@webtide.com> (@gregw)
```

Comment:
Looks like this line broke the build in Travis.

Reply:
oops indent is missing? fixing...